docs/api.rst

```diff
@@ -19,7 +19,7 @@ Base Endpoint API
 AsyncioEndpoint
 ---------------
 
-.. automodule:: lahja.endpoint.asyncio.endpoint
+.. autoclass:: lahja.asyncio.endpoint.AsyncioEndpoint
```

Comment:
Can you add the `lahja.trio.endpoint.TrioEndpoint`?

Reply:
Added :+1: